kong/cmd/start.lua

```diff
@@ -25,6 +25,9 @@ end
 local lapp = [[
 Usage: kong start [OPTIONS]
 
+Start Kong (Nginx and other configured services) in configured
```

Comment:
Gave some thoughts about this but "dependencies" can be anything, and I originally wanted to mention "daemons", considering Serf and dnsmasq are spawned in the background, hence why "services", to carry the meaning that Kong spawns multiple daemons...